packages/manager/apps/container/src/container/nav-reshuffle/header/user-account-menu/Content.tsx

```diff
@@ -167,15 +169,14 @@ const UserAccountMenu = ({
               className={`d-flex mt-1 justify-content-between ${style.menuContentRow}`}
             >
               <span>{t('user_account_menu_support')}</span>
-              <a href={supportLink}>
+              <a href={supportLink} onClick={() => onTrackNavigation(tracking.supportLevel)}>
```

Comment:
`supportLink` is not the correct one for EU and CA.
They need to redirect to help center